samtranslator/internal/schema_source/aws_serverless_graphqlapi.py

```diff
@@ -152,7 +152,19 @@ class Cache(BaseModel):
     TransitEncryptionEnabled: Optional[PassThroughProp]
 
 
+class Defaults(BaseModel):
+    # properties common for Resolvers and Functions
+    Runtime: Optional[Runtime]
+    MaxBatchSize: Optional[PassThroughProp]
+    Sync: Optional[Sync]
```

Comment:
Is it guaranteed `Sync` will always be exactly the same for resolvers and functions?

Reply:
There will be no Sync in new version